.circleci/config.yml

```diff
@@ -45,23 +55,50 @@ commands:
           steps:
             - run: apt-get update -y && apt-get install -y make
 
-      - run: make install-deps EXECUTOR=<< parameters.os >>
+      - when:
+          condition:
+            not:
+              equal: [ "win/default", << parameters.os >> ]
+          steps:
+            - run: make install-deps EXECUTOR=<< parameters.os >>
+
+      - when:
+          condition:
+            equal: [ "win/default", << parameters.os >> ]
+          steps:
+            - run: choco install cmake.install --installargs '"ADD_CMAKE_TO_PATH=User"'
+            - run: choco install ninja
 
   build:
+    parameters:
+      <<: *common_parameters
     steps:
-      - run:
-          # NOTE: the circle ci container executor reports the memory/cpu stats
-          # of the host machine (https://ideas.circleci.com/ideas/CCI-I-578),
-          # `nproc` will return 36 on docker/medium resource_class,
-          # ninja parallelizes accordingly, which leads to overloading
-          # and circleci eventually terminating the builds:
-          #
-          # ninja: job failed: /usr/bin/c++ [...]
-          # c++: fatal error: Killed signal terminated program cc1plus
-          # compilation terminated
-          #
-          # force overwrite job count here:
-          command: make lief JOBS=3
+      - when:
+          condition:
+            not:
+              equal: [ "win/default", << parameters.os >> ]
+          steps:
+            - run:
+                # NOTE: the circle ci container executor reports the memory/cpu stats
+                # of the host machine (https://ideas.circleci.com/ideas/CCI-I-578),
+                # `nproc` will return 36 on docker/medium resource_class,
+                # ninja parallelizes accordingly, which leads to overloading
+                # and circleci eventually terminating the builds:
+                #
+                # ninja: job failed: /usr/bin/c++ [...]
+                # c++: fatal error: Killed signal terminated program cc1plus
+                # compilation terminated
+                #
+                # force overwrite job count here:
+                command: make lief JOBS=3
+
+      - when:
+          condition:
+            equal: [ "win/default", << parameters.os >> ]
+          steps:
+            - run:
+               command: refreshenv & cd vendor\lief & python ./setup.py --ninja build_ext -b ..\..\dist\lief
```

Comment:
Yes. I didn't see an easy way to take that kind of route (busybox, cygwin, etc) without significantly increasing the complexity of the CircleCI config. I'd prefer to stick with this approach for now, since the build process is going to change in the (hopefully near) future anyway.